docs/webserver.md

```diff
@@ -0,0 +1,51 @@
+# Webserver
+
+| | web02.rackspace.theforeman.org |
+| - | - |
+| type | OpenStack VM |
+| OS | CentOS 7 |
+| CPUs | 2 |
+| RAM | 2GB |
+| Storage | /dev/xvda (40GB): root, /dev/xvdb (100GB) + /dev/xvdc (50GB): data |
+| Managed by | [web.pp](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/profiles/manifests/web.pp) |
+
+## Domains
+
+These domains are all hosted on the webserver.
+
+* theforeman.org, www.theforeman.org
+* deb.theforeman.org
+* debugs.theforeman.org
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+* yum.theforeman.org
+* rsync.theforeman.org
+
+### Fastly CDN
+
+A Fastly CDN exists that sits in front of:
+
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+* yum.theforeman.org
+
+For these, the webserver acts as a backend while the content is served from the Fastly CDN to users.
```

Comment:
There is also https://projects.theforeman.org/projects/foreman/wiki/Infrastructure_CDN which is a tad outdated and there is a bit of ansible to deploy that setup in `foreman-infra/ansible/` (mostly undocumented)

I wonder if we should fold that completely into webserver.md (I can do as soon as we merge this PR)?